x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/suggestions_callout.tsx

```diff
@@ -28,13 +28,15 @@ interface SuggestionsCalloutProps {
   description: string;
   buttonTo: string;
   lastUpdatedTimestamp: string; // ISO string like '2021-10-04T18:53:02.784Z'
+  style?: React.CSSProperties;
```

Comment:
For a minute I had an implementation of this that added optional `appendSpacer` and `prependSpacer` booleans to `SuggestionsCalloutProps`.  Inline styles are bad but I think single use props are worse...  alternatively I could stick this stuff in a pair of scss files? We're not super consistent about doing one-off css styles.